public/manifest.webmanifest

```
@@ -0,0 +1,53 @@
+{
+  "name": "Bellshade - Learn with Our Community",
+  "short_name": "Bellshade",
+  "description": "Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia",
+  "theme_color": "#ffffff",
+  "background_color": "#ffffff",
+  "display": "standalone",
+  "orientation": "any",
+  "scope": "/",
+  "start_url": "/",
+  "icons": [
+    {
+      "src": "icons/icon-72x72.png",
+      "sizes": "72x72",
+      "type": "image/png"
```

```suggestion
      "type": "image/png",
      "purpose": "any maskable"
```

ditambahin ` "purpose": "any maskable"` di setiap size gambarnya. Sekalian diganti nama file yang ini jadi `manifest.json` sesuai yang sebelumnya